actions/st2_upgrade_to_enterprise.sh

```diff
@@ -20,22 +21,56 @@ if [ "${RELEASE}" = "unstable" ]; then
     REPO="${REPO}-${RELEASE}"
 fi
 
-# Install enterprise bits
-echo "Downloading from repo ${REPO}..."
-
-if [[ ${DISTRO} = \UBUNTU* ]]; then
-    curl -s https://${LICENSE_KEY}:@packagecloud.io/install/repositories/StackStorm/${REPO}/script.deb.sh | bash
-    apt-get install -y st2flow
-    apt-get install -y st2-auth-ldap
-else
-    curl -s https://${LICENSE_KEY}:@packagecloud.io/install/repositories/StackStorm/${REPO}/script.rpm.sh | sudo bash
-    yum install -y st2flow
-    yum install -y st2-auth-ldap
-fi
+get_apt_pkg_latest_revision() {
+    # Returns string of form 1.5.1-5. Note how this is different from rpm :/
+    local PKG_NAME=$1
+    local PKG_VERSION=$2
+
+    apt-cache show $PKG_NAME | grep Version | awk '{print $2}' | grep $PKG_VERSION | sort --version-sort | tail -n 1
+}
+
+get_rpm_pkg_name_with_latest_revision() {
+    # Returns string of form st2-0:1.5.1-3.x86_64
+    local PKG_NAME=$1
+    local PKG_VERSION=$2
+
+    yum install -y yum-utils # need repoquery
+    repoquery --show-duplicates $PKG_NAME-$PKG_VERSION* | sort --version-sort | tail -n 1
+}
 
-# Replace the auth section
-echo "Updating st2.conf..."
-sed -i -e '/\[system\]/ p' -e '/\[auth\]/,/\[system\]/ d' /etc/st2/st2.conf
+install_enterprise_bits() {
+    # Install enterprise bits from packagecloud
+    echo "Downloading from repo ${REPO}..."
+
+    if [[ ${DISTRO} = \UBUNTU* ]]; then
+        curl -s https://${LICENSE_KEY}:@packagecloud.io/install/repositories/StackStorm/${REPO}/script.deb.sh | bash
+        if [[ -z $VERSION ]]; then
+            apt-get install -y st2flow
+            apt-get install -y st2-auth-ldap
```

Comment:
Is the st2enterprise package still broken or should we switch to that instead of the individual packages?